src/frontend/src/lib/stores/identity-info.state.svelte.ts

```diff
@@ -141,6 +145,20 @@ class IdentityInfo {
     const googleRemoveResult = await googleRemovePromise;
 
     if ("Ok" in googleRemoveResult) {
+      // If we just deleted the method we are logged in with, we log the user out.
+      const lastUsedAuthMethod = get(lastUsedIdentityStore)?.authMethod;
+      if (
+        lastUsedAuthMethod &&
+        "openid" in lastUsedAuthMethod &&
+        lastUsedAuthMethod.openid.sub === temporaryCredential.sub
```

Comment:
How about creating the variable `isOpenIdCurrentAccessMethod` before we make the call to google and remove it so that we can use it here instead of all these lines? You can take the value from the method `isOpenIdCurrentAccessMethod` that I propose to add to `identityInfo` as well.